tests/wpt/metadata/MANIFEST.json

```diff
@@ -33717,7 +33717,22 @@
   },
   "local_changes": {
     "deleted": [],
-    "items": {},
+    "items": {
+      "testharness": {
+        "websockets/Send-data-worker.htm": [
+          {
+            "path": "websockets/Send-data-worker.htm",
```

Comment:
I don't think this should be here.

r+ assuming you file a followup about checking the Origin header in workers.